src/common/options.cc

```diff
@@ -1532,6 +1532,11 @@ std::vector<Option> get_global_options() {
     .add_service("mon")
     .set_description("maximum time for a mon election (seconds)"),
 
+    Option("mon_election_default_strategy", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(1)
+    .set_min_max(1, 3)
```

Comment:
maybe make this an enum instead?

Reply:
Is there a way to do that in Option? It's enums in the rest of the codebase.

Reply:
see for example osd_pool_default_type.  the underlying type is a string, but the values are constrained.

Reply:
Hmm that's cool but I'd have to add more string->int parsing code and I really don't anticipate anybody except a Ceph dev setting this option.